### DISCUSSION TITLE
How to find broken links with fd?

### DISCUSSION THREAD

**Discussion by davecoutts** [sharkdp/fd#1298] [General]
I would like to find broken links ... fast, using ```fd```.

Is there an equvilent to ```find /blah -xtype -l``` in ```fd```?

**Comment by sharkdp** [comment]
We don't really support this as a special feature, but you can actually use the `-tl`/`--type=symlink`option to discover broken symlinks if you combine it with `-L`/`--follow`:
```bash
fd --type=symlink --follow
```
or in short:
```bash
fd -tl -L  # or even fd -Ltl
```

But note that this will follow *valid* symlinks while searching. You can combine this with other options such as `--hidden`, `--no-ignore`, etc.

**Reply by davecoutts** [reply]
Thanks for the guidance David. I will give -Ltl a try.

**Reply by UnkwUsr** [reply]
@sharkdp I believe this should be documented (probably in description of `-L` flag, that it changes behavior of `-tl`) (I'm not rising PR as I'm not feeling my language skill good enough for writing docs :( )
